### ISSUE TITLE
Rewrite item tooltip

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#70]
Use a div with absolute positioning to generate a new tooltip.

Add fame bonus, bag type, ut/st status to tooltip data.

This issue is dependent on #69.

**Comment by tuvior** [comment]
new format: 
```javascript
type: ["id", SlotType, Tier, x, y, FameBonus, feedPower, BagType, Soulbound, UT/ST],
```
for the last index, default: 0, UT:1, ST:2

**Comment by jakcodex** [comment]
Released in 0.8.3-0.
